packages/expo-dev-launcher/babel.config.js

```diff
@@ -31,6 +33,17 @@ module.exports = function (api) {
     alias['react-native-safe-area-context'] = safeAreaContext;
   }
 
+  // manually resolve react-navigation packages -- remove when we have updated react-navigation to ~6
+  alias['@react-navigation/bottom-tabs'] = tryResolveModule(
```

Comment:
@lukmccall @esamelson @FiberJW had to do this to get the module resolution to work - do you know if there's a better way?